backend/test/unit/controller/chatController.test.ts

```diff
@@ -741,55 +739,6 @@ describe('handleChatToGPT unit tests', () => {
 	});
 });
 
-describe('handleGetChatHistory', () => {
```

Comment:
As you've combined the logic into levelController, and deleted these tests, shouldn't you be replacing them with a test suite for the new controller, rather than just deleting them?

Reply:
done